ocw/lib/azure.py

```diff
@@ -206,3 +206,14 @@ def cleanup_gallery_img_versions(self) -> None:
                         self.compute_mgmt_client().gallery_image_versions.begin_delete(
                                 self.__resource_group, gallery.name, image.name, version.name
                         )
+
+    def get_img_versions_count(self) -> int:
+        self.log_dbg("Call get_img_versions_count")
+        gallery = self.compute_mgmt_client().galleries.get(self.__resource_group, self.__gallery)
+        all_img_versions = 0
+        for image in self.compute_mgmt_client().gallery_images.list_by_gallery(self.__resource_group, gallery.name):
+            img_version = len(list(self.compute_mgmt_client().gallery_image_versions.list_by_gallery_image(
```

Comment:
I would rename this variable because right now it seems like holding a version, instead of the number of image versions.

```suggestion
            img_versions = len(list(self.compute_mgmt_client().gallery_image_versions.list_by_gallery_image(
```